dev/NavigationView/NavigationView_ApiTests/NavigationViewTests.cs

```diff
@@ -477,6 +477,33 @@ public void VerifySettingsItemToolTip()
             });
         }
 
+        [TestMethod]
+        public void VerifySettingsItemTag()
+        {
+            NavigationView navView = null;
+            NavigationViewItem settingsItem = null;
+
+            RunOnUIThread.Execute(() =>
+            {
+                navView = new NavigationView();
+
+                navView.IsSettingsVisible = true;
+                navView.IsPaneOpen = true;
+                navView.PaneDisplayMode = NavigationViewPaneDisplayMode.Left;
+                MUXControlsTestApp.App.TestContentRoot = navView;
+            });
+
+            IdleSynchronizer.Wait();
+
+            RunOnUIThread.Execute(() =>
+            {
+                settingsItem = (NavigationViewItem)navView.SettingsItem;
+                Verify.AreEqual(settingsItem.Tag, "Settings");
+
+                MUXControlsTestApp.App.TestContentRoot = null;
```

Comment:
Is this line required?

Reply:
Some test class has a cleanup function (which cleans up the TestContentRoot) that executes after each tests run, I noticed NavigationViewTests doesn't have it so I added that line here.

It'll probably be ok too without this line, the next test will just override the content, but I guess it's better to keep it there so we leave a good and clean state for next test.